api/datastore/postgres/postgres.go

```diff
@@ -185,17 +173,18 @@ func (ds *PostgresDatastore) GetApp(ctx context.Context, name string) (*models.A
 	var resName string
 	var config string
 	err := row.Scan(&resName, &config)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, models.ErrAppsNotFound
+		}
+		return nil, err
+	}
 
```

Comment:
I would like to wrap most of the errors (the random, non-static ones) across the Datsatore with more context, but I didn't want to break local style too much.